sale_order_type/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+* Go to **Sales > Sales Orders** and create a new sale order. Select the new type you have created before and all settings will be propagated.
+* You can also define a type for a particular partner if you go to *Sales & Purchases* and set a sale order type.
```

Comment:
Add a point about the new `find by rule` button

Reply:
Done in https://github.com/OCA/sale-workflow/pull/716/commits/148e7b60af81508b48788a45021d44f6d8e15ca3